setup.cfg

```diff
@@ -23,3 +23,4 @@ skip = httpcore/_sync/,tests/sync_tests/
 addopts = --cov-report= --cov=httpcore --cov=tests -rxXs
 markers =
   copied_from(source, changes=None): mark test as copied from somewhere else, along with a description of changes made to accodomate e.g. our test setup
+  curio: mark the test as a coroutine, it will be run using a Curio kernel.
```

Comment:
I _think_ there's a programmatic way to tell pytest "mark this mark as known" from one of the pytest hooks, though I don't exactly remember which one. Might be preferable to keep the curio pytest integration self-contained in a single file (and so easy to remove eg if/when we move to anyio).

Reply:
There — found this programmatic method here: https://docs.pytest.org/en/stable/example/markers.html#custom-marker-and-command-line-option-to-control-test-runs

```python
def pytest_configure(config):
    # register an additional marker
    config.addinivalue_line(
        "markers", "env(name): mark test to run only on named environment"
    )
```

Reply:
Thank you, I added that to `test/conftest.py` in https://github.com/encode/httpcore/pull/168/commits/5c1e69bbf8ee8be31beb391bc76f6e1a5a92aa75 . So that the "curio" marker is discoverable using `pytest --markers`:
```
% pytest --markers | grep curio
@pytest.mark.curio: mark the test as a coroutine, it will be run using a Curio kernel.
```